app/src/main/res/values/strings.xml

```diff
@@ -1496,6 +1496,11 @@
     <string name="preferences_addresses_add_address">Add address</string>
     <!-- Preference option for managing saved addresses -->
     <string name="preferences_addresses_manage_addresses">Manage addresses</string>
+    <!-- Preference for saving and autofilling addresses -->
+    <string name="preferences_addresses_save_and_autofill_addresses">Save and autofill addresses</string>
+    <!-- Preference summary for saving and autofilling address data -->
+    <string name="preferences_addresses_save_and_autofill_addresses_summary">Addresses summary</string>
```

Comment:
I think you used a placeholder string here. I don't know if the strings are finalized so maybe you should also double check in the address group chat, but otherwise strings are at https://www.figma.com/file/cSKRUEvfJSzNBv2Mp65Pdv/Mobile-CC-Autofill?node-id=1356%3A36515

Reply:
The string is misspelled in Figma, so I assumed it's not finalized yet.